core/controllers/story_editor.py

```diff
@@ -144,14 +155,31 @@ class StoryPublishHandler(base.BaseHandler):
     """A data handler for publishing and unpublishing stories."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'story_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'PUT': {
+            'new_story_status_is_public': {
+                'schema': {
+                    'type': 'bool'
+                },
+                'default_value': None
```

Comment:
Why is default value `None`?

Reply:
There was default_value at some other places so I added it here

Reply:
Does it make sense here? What was the behaviour before if this thing wasn't set?